### ISSUE TITLE
Sankey: Failed to execute 'getComputedStyle' on 'Window'

### ISSUE THREAD

**Issue by asterikx** [f5/unovis#123]
When I deploy the [Basic Sankey](https://unovis.dev/gallery/view/?collection=Networks%20and%20Flows&title=Basic%20Sankey) in a Next.js application, I get the following error:

```
Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at getCSSVariableValue (misc.js:36:12)
    at getCSSVariableValueInPixels (misc.js:39:17)
    at get bleed [as bleed] (index.js:75:167)
    at Sankey._render (index.js:123:25)
    at Sankey.render (index.js:56:14)
    at SingleContainer._render (index.js:82:19)
    at eval (index.js:63:22)
```

Reproduction: [Link](https://stackblitz.com/edit/github-mmbfee-siably?file=pages/index.tsx)

**Comment by rokotyan** [comment]
@asterikx Next.js throws an error because it tries to compile the app on the server side but Unovis doesn't support Server Side Rendering.

Usually workarounds like [these](https://blog.bitsrc.io/using-non-ssr-friendly-components-with-next-js-916f38e8992c) make it work. I've tried a few of them today they didn't solve the issue. I'll take a deeper look later and let you know.

**Comment by rokotyan** [comment]
@asterikx I've tried to initialize a new Next.js app and Unovis worked there out of the box. You can use that as a temporary solution. I'm still not sure why your example doesn't work even when SSR is disabled.

**Comment by asterikx** [comment]
@rokotyan That's not really an option.

However, I can workaround the issue by passing `config.labelFontSize`:

https://github.com/f5/unovis/blob/555e78dab536a89224713cf3a3ff4ee4f5565ca1/packages/ts/src/components/sankey/index.ts#L79

**Comment by asterikx** [comment]
@rokotyan  I have another problem with the Sankey component when wrapping it. I'm not sure if it's possibly related to this one.

When the wrapped Sankey is re-rendered it throws an error:

```
react.js?13db:12 Uncaught TypeError: Cannot read properties of null (reading 'props')
    at arePropsEqual (react.js?13db:12:26)
    at updateMemoComponent (react-dom.development.js?ac89:19329:1)
    at beginWork (react-dom.development.js?ac89:21673:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js?ac89:4164:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?ac89:4213:1)
    at invokeGuardedCallback (react-dom.development.js?ac89:4277:1)
    at beginWork$1 (react-dom.development.js?ac89:27451:1)
    at performUnitOfWork (react-dom.development.js?ac89:26557:1)
    at workLoopSync (react-dom.development.js?ac89:26466:1)
    at renderRootSync (react-dom.development.js?ac89:26434:1)
    at performSyncWorkOnRoot (react-dom.development.js?ac89:26085:1)
    at flushSyncCallbacks (react-dom.development.js?ac89:12042:1)
    at eval (react-dom.development.js?ac89:25651:1)
```

Maybe there is an unbound `this` in the `VisSankey` component (or somewhere else in the callstack)?

[Link](https://stackblitz.com/edit/github-mmbfee-i5w15y?file=pages/index.tsx) to reproduction.

**Comment by rokotyan** [comment]
@asterikx That's a different issue, thanks for reporting. It's fixed in version `1.0.3-beta.3`.
